tests/test_agile_sprints.py

```
@@ -0,0 +1,216 @@
+        assert s.title == "Login"
+        assert s.story_points == 3
+        assert s.status == StoryStatus.BACKLOG
+
+    def test_default_points(self):
+        s = UserStory(story_id="s1", title="x")
+        assert s.story_points == 1
+
+    def test_negative_points_raises(self):
+        try:
+            UserStory(story_id="s1", title="x", story_points=-1)
+            assert False, "Expected ValueError"
```

Ruff (0.15.15)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)